sources/vcdMerge/test/functional/test.xml

```diff
@@ -234,5 +234,202 @@ A sample test definition (containing all tags):
         </sources>
     </unique>
   </merge>
+  
+  <!-- User log format tests.-->
+  <merge>
+    <info>
+        <name>Smoke test 6</name>
+        <description>A test with the output in the default time units. Not synchronized sources. User log format.</description>
+    </info>
+    <common>
+        <verbose>False</verbose>
+        <output_file>ul_test0_us.output</output_file>
+        <golden_file>test0_us.vcd</golden_file>
+    </common>
+    <unique>
+        <sources>
+            <source>
+                <format>U{../common/doc/examples/vcdMaker.xml}</format>
```

Comment:
I think tests should use their own xml file. Even if it's currently the same as example.
Test xml should be in the directory with the tests (or in new dir under tests, in case we would want to have more that one xml)

Reply:
Sounds reasonably. I will add the file to common/test. It will to common for the vcdMaker and vcdMerge.